qt/python/mantidqt/mantidqt/_common.sip

```diff
@@ -1029,6 +1029,7 @@ class ImageInfoWidget : public QTableWidget {
 public:
     ImageInfoWidget(QWidget *parent = nullptr);
     void cursorAt(const double x, const double y, const double signal, const QMap<QString, QString>& extraValues) throw(std::exception);
+    void setShowSignal(const bool showSignal);
```

Comment:
```suggestion
    void setShowSignal(const bool showSignal) throw(std::exception);
```
Does this need a `throw` marker? The other methods for this object seem to have one.

Reply:
So I've always assumed that you mark something with `throw(std::exception)` in a sip export if that function is expected to throw an exception? But I'm actually not sure if that's what that means.